CRM/Upgrade/Incremental/php/FiveTwentySix.php

```diff
@@ -26,9 +26,14 @@ class CRM_Upgrade_Incremental_php_FiveTwentySix extends CRM_Upgrade_Incremental_
    */
   public function setPreUpgradeMessage(&$preUpgradeMessage, $rev, $currentVer = NULL) {
     // Example: Generate a pre-upgrade message.
-    // if ($rev == '5.12.34') {
-    //   $preUpgradeMessage .= '<p>' . ts('A new permission, "%1", has been added. This permission is now used to control access to the Manage Tags screen.', array(1 => ts('manage tags'))) . '</p>';
-    // }
+    if (!function_exists('civi_wp')) {
+      //exit
+    }
+    elseif ($rev == '5.26.alpha1') {
```

Comment:
How does this work, will it show ONLY for 5.26.alpha1? Will it still show when a WP user tries to upgrade to 5.26.0, 5.27?

Reply:
@MikeyMJCO My first pre/post upgrade but since they fire in sequence if someone goes from 5.21 to 5.27 this will alert them

Reply:
So if they try to upgrade from 5.26.0 then no as this file won't be run in that upgrade, if they try to upgrade from 5.25.0 then yes it would because the upgrader has to go through the alpha1 and beta1 stages.